dash/development/base_component.py

```diff
@@ -378,7 +378,7 @@ def __repr__(self):
           '{:s}=Component.UNDEFINED'.format(p))
          for p in prop_keys
          if not p.endswith("-*") and
```

Comment:
Could also fix the import problem with dash_flow_example by changing to `p.endswith('*')`, but I guess it should be changed in the `dash_flow_example` repo to a valid wildcard attribute like you proposed in #319.